backend/server/mongodb/models/TrainingLog 2.ts

```diff
@@ -0,0 +1,43 @@
+import mongoose, { Schema } from "mongoose";
+import { TrainingLog, ServiceAnimalBehavior } from "src/utils/types";
+
+const ServiceAnimalBehaviorSchema = new mongoose.Schema<ServiceAnimalBehavior>({
+  description: String,
+  repeat: Number,
+});
+
+const TrainingLogSchema = new mongoose.Schema<TrainingLog>({
+  date: {
+    type: Date,
+    required: true,
+    index: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  skills: {
+    type: [String],
+    required: true,
+  },
+  trainingHours: {
+    type: Number,
+    required: true,
+  },
+  behavior: {
+    type: ServiceAnimalBehaviorSchema,
+    required: true,
+  },
+  animal: {
+    type: Schema.Types.ObjectId,
+    required: true,
+  },
+  video: {
+    type: String,
+  },
+});
+
+const TrainingLogModel =
+  (mongoose.models.TrainingLog as mongoose.Model<TrainingLog>) ||
+  mongoose.model<TrainingLog>("TrainingLog", TrainingLogSchema);
+export default TrainingLogModel;
```

Comment:
once again, not sure why the models are replicated too (can you delete these replicated files too?)